<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolTreeWidget</class>
 <widget class="QWidget" name="SymbolTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomPanel">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterBox">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCopyName">
   <property name="text">
    <string>Copy Name</string>
   </property>
  </action>
  <action name="actionCopyLocation">
   <property name="text">
    <string>Copy Location</string>
   </property>
  </action>
  <action name="actionGoToInDisassembly">
   <property name="text">
    <string>Go to in Disassembly</string>
   </property>
  </action>
  <action name="actionGoToInMemoryView">
   <property name="text">
    <string>Go to in Memory View</string>
   </property>
  </action>
  <action name="actionGroupBySection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by section</string>
   </property>
  </action>
  <action name="actionGroupBySourceFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by source file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
